### DISCUSSION TITLE
S7 commands for Giesen

### DISCUSSION THREAD

**Discussion by PhilippSchoenbaechler** [artisan-roaster-scope/artisan#827] [Q&A] [answered]
Hello

I would like to be able to control more functions from Artisan on our Giesen.
Does anyone have a list of all S7 commands that are possible?

I am particularly interested in the following questions/functions:
1. An offset of 1 must be applied for the Burner setting. For 10% the command looks like this: setDBint(2,14,11).
Is there any way to eliminate this offset?
2.  I would also like to be able to start the exhaust air from the cooling screen and the agitator from Artisan. Does anyone know the commands for this?

Thank you much for any inputs/help.

Best wishes,
Philipp

**Comment by MAKOMO** [comment, marked as answer]
If you type COMMAND-2 (CTR-2 on Windows) you switch to the palette defining an extended set of custom buttons on the official Giesen setups in Artisan v2.4.6 that contains buttons for starting and stopping the stirrer on supported machines (note that each machine runs a slightly different firmware and connects different actors). See https://artisan-scope.org/machines/giesen/. All known commands are defined here, others might be possible, like turning the roasting light on and off. For those you need to reengineer the machine or ask Giesen. Good luck on both!

The burner offset is build into the machine. Setting the burner to 0 activates the automatic mode on those machines. A design by Giesen.
